packages/smooth_app/README.md

```diff
@@ -2,13 +2,15 @@
 
 # Open Food Facts - Codename: "Smooth App"
 
-A new Flutter application by [Open Food Facts](https://github.com/openfoodfacts). You can install it on [Android](https://play.google.com/store/apps/details?id=org.openfoodfacts.app) or [iPhone/iPad](https://apps.apple.com/us/app/smooth-app/id1526747703). Note that a internal development build ([Android](https://play.google.com/apps/internaltest/4700279390303733107) or [iPhone/iPad](https://testflight.apple.com/join/dIhF6Gi4) )if you'd like to use the results of your PRs quicker.
+A new Flutter application by [Open Food Facts](https://github.com/openfoodfacts) that can be installed on [Android](https://play.google.com/store/apps/details?id=org.openfoodfacts.app) or [iOS](https://apps.apple.com/us/app/smooth-app/id1526747703) based devices. Note that a internal development build ([Android](https://play.google.com/apps/internaltest/4700279390303733107) or [iPhone/iPad](https://testflight.apple.com/join/dIhF6Gi4) )if you'd like to use the results of your PRs quicker.
```

Comment:
You can also a space before the if: "dIhF6Gi4) )if you'd"

Reply:
Okay. Gonna do that now.